improve choice of vurb (#377)
publiccodenet/about: contributor-guides/for-staff.md

---
type: Guide

> explains: How to approach the requirements of your employment with the Foundation for Public Code

Reviewer: ```suggestion
explains: The basics for being a staff member of the Foundation for Public Code
```
